<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XorCipher</class>
 <widget class="QWidget" name="XorCipher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="messageLabel">
     <property name="text">
      <string>Исходный текст</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="keyLabel">
     <property name="text">
      <string>Гамма-последовательность</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="messageEdit"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="keyEdit"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="codeLabel">
     <property name="text">
      <string>Криптограмма</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="decodeButton">
     <property name="text">
      <string>Расшифровать</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="generatorBox">
     <property name="title">
      <string>Линейный конгруэнтный генератор ПСЧ</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="seedLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Начальное значение</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="multiplierSpin">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="seedEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="maxLength">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="incrementSpin">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="multiplierLabel">
        <property name="text">
         <string>Множитель</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="incrementLabel">
        <property name="text">
         <string>Приращение</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="generateKeyButton">
     <property name="text">
      <string>Сгенерировать</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="codeEdit"/>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="encodeButton">
     <property name="text">
      <string>Зашифровать</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
